lightnode/chunkgroup/calculations.go

```diff
@@ -0,0 +1,115 @@
+package chunkgroup
+
+import (
+	"fmt"
+	"golang.org/x/crypto/sha3"
+	"time"
+)
+
+// uint64ToBytes converts a uint64 to a byte array in big-endian order.
+func uint64ToBytes(value uint64) []byte {
+	bytes := [8]byte{}
+	for i := 0; i < 8; i++ {
+		bytes[i] = byte(value >> (56 - 8*i))
+	}
+	return bytes[:]
+}
+
+// bytesToUint64 converts a byte array to a uint64 in big-endian order.
+func bytesToUint64(bytes []byte) uint64 {
+	var value uint64
+	for i := 0; i < 8; i++ {
+		value |= uint64(bytes[i]) << (56 - 8*i)
+	}
+	return value
+}
+
+// hash hashes a seed using the Keccak-256 algorithm. The returned value is an integer formed from the
+// first 8 bytes of the hash.
+func randomInt(seed uint64) uint64 {
+	hasher := sha3.NewLegacyKeccak256()
```

Comment:
@mooselumph Is this the correct function to call for a standard keccak hash, or should I instead be calling `sha3.New256()'?